students/jim_krecek/lesson04/mailroom_p2.py

```diff
@@ -44,16 +42,17 @@ def send_all():
     print("\nLetters have been printed and are saved in the current directory")
     
 def main():
+    main_menu = {'1': send_ty,'2': create_report,'3':send_all,'4':False}
```

Comment:
map `4` to quit program - the advantage of using switch_dict is how simple your main loop becomes

```
def quit_program():
    print("bye")
    sys.exit()

    main_menu = {'1': send_ty,'2': create_report,'3':send_all,'4': quit_program}
```